apps/docs/components/edit-feedback.tsx

```diff
@@ -0,0 +1,37 @@
+type MinimalPost = {
+  title: string
+  slug: string
+}
+
+type Props = {
+  post: MinimalPost
+  contentPath: string
+}
+
+export default function EditFeedbackLinks({ post, contentPath }: Props) {
+  const pageEditUrl = `https://github.com/IHIutch/draft-ui/tree/main/apps/docs/content/${contentPath}`
+  const pageFeedbackUrl = new URL(
+    `https://github.com/IHIutch/draft-ui/issues/new`,
+  )
+  pageFeedbackUrl.searchParams.set('title', `Feedback for “${post.title}”`)
+  // Is this the right label to set? Do we need to make a new label for this?
+  pageFeedbackUrl.searchParams.set('labels', 'documentation')
+
+  return (
+    <span className="flex flex-col space-y-2 pt-6">
```

Comment:
Couple notes here:
- Let's use a list for the links here. Semantically, I'm not entirely sure it matters, but that's what we do for the ToC and I'd rather opt for consistency if nothing else. (In general, I'd avoid converting spans to non-inline elements. This has the potential to create invalid HTML if you wanted to make your links display: block, for example)
- Let's also use the link style/spacing from the ToC links for consistency